source/hwIo/base.py

```diff
@@ -10,37 +10,43 @@
 See L{braille.BrailleDisplayDriver.isThreadSafe}.
 """
 
+from __future__ import annotations

```

Comment:
same here?

Reply:
It is otherwise not possible to annotate with `weakref.ReferenceType[IoThread]`. This will be possible in newer versions of python.

Reply:
Maybe adding a comment indicating what it is for and also indicating what will be possible in future versions of Python, so that, when upgrading Python, this import can be cleaned up.

Reply:
Yes please add comments on the future imports, thanks @leonardder